packages/block-editor/src/components/inserter/hooks/use-insertion-point.js

```diff
@@ -119,10 +120,36 @@ function useInsertionPoint( {
 					meta
 				);
 			} else {
+				const normalizedBlocks = Array.isArray( blocks )
+					? blocks
+					: [ blocks ];
+
+				let _destinationIndex = destinationIndex;
+				let _destinationRootClientId = destinationRootClientId;
+				// Check if it's possible to insert the block in the current location
+				const canInsertBlocks = normalizedBlocks.every( ( block ) =>
+					canInsertBlockType( block.name, _destinationRootClientId )
+				);
+
+				if ( ! canInsertBlocks && normalizedBlocks.length === 1 ) {
```

Comment:
Why is this only if the block length is 1? What if we're inserting a group of blocks?

Reply:
For now I think we should default to this case until it's ironed out.